.ci/validation.sh

```diff
@@ -1074,6 +1074,15 @@ assembly-site)
   mvn -e --no-transfer-progress site -Dlinkcheck.skip=true
   ;;
 
+yamllint)
+  sudo apt update
+  sudo apt install -y yamllint
```

Comment:
Environment preparation (package installation) is always outside, as it different even in different OS.

We usually keep it in CI config, close to definition of docker image or OS for a job

Reply:
done